src/Sentry.AspNetCore/ApplicationBuilderExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static IApplicationBuilder UseSentry(this IApplicationBuilder app)
                 var stackTraceFactory = app.ApplicationServices.GetService<ISentryStackTraceFactory>();
                 if (stackTraceFactory != null)
                 {
-                    _ = o.UseStackTraceFactory(stackTraceFactory);
+                    o.UseStackTraceFactory(stackTraceFactory);
```

Comment:
WHat about a find and replace on the whole project if this will be a thing now?
Better than noise on each PR.

Also editorconfig rule to warn to add this has to go if it's not yet gone

Reply:
I have never had a warning about this, so I presume there is no rule in editorconfig, or it's just not enforced.
I will search & replace in another PR.

Reply:
https://github.com/getsentry/sentry-dotnet/blob/54af283fa2c788e5416c23dbba5f55c6698c5b94/.editorconfig#L120